hw/ip/flash_ctrl/data/flash_ctrl.hjson.tpl

```diff
@@ -887,6 +887,8 @@
           desc: "Trigger flash phy alert"
         }
       ]
+      tags: [ // alert triggers should be tested by directed tests
```

Comment:
Tags can be added per-CSR field as well. I suppose we only want to prevent the write to `alert_trig` field?